ibis/formats/__init__.py

```diff
@@ -168,15 +168,13 @@ def convert_column(cls, obj: C, dtype: DataType) -> C:
         raise NotImplementedError
 
     @classmethod
-    def convert_table(cls, obj: T, schema: Schema) -> T:
+    def convert_table(cls, obj: T) -> T:
```

Comment:
This now seems like a nonsensical function.  Convert it how?  and to what?  

If every child class that uses this defines a schema argument, shouldn't we keep it here?

Reply:
Good catch, seems like this should stay.

Reply:
I can put up a PR to restore it

Reply:
https://github.com/ibis-project/ibis/pull/8821